tests/reflow.js

```diff
@@ -1,4 +1,5 @@
 import StyleObserver from "../index.js";
+import adoptCSS from "../src/util/adopt-css.js";
```

Comment:
```suggestion
import gentleRegisterProperty from "../src/util/gentle-register-property.js";
```

Reply:
This one will register with `*` if the registration fails. For the tests we need the other one (see above)

Reply:
Oh, right! Fixed the suggestion.